website/content/docs/commands/init.mdx

```diff
@@ -32,10 +15,21 @@ that are named `packer-plugin-*` -- to install a single plugin binary -- that is
 `packer-provisioner-*`, `packer-builder-*`, etc. -- nothing changes, you will
 have to [install the plugin manually](/docs/plugins#installing-plugins).
 
-The `packer init` command is used to download Packer plugin binaries.
-This is the first command that should be executed when working with a new or
-existing template. This command is always safe to run multiple times. Though
-subsequent runs may give errors, this command will never delete anything.
+The `packer init` command is used to download Packer plugin binaries. This is
+the first command that should be executed when working with a new or existing
+template. This command is always safe to run multiple times. Though subsequent
+runs may give errors, this command will never delete anything. Packer does not
+currently have the notion of a state like Terraform has. In other words,
+currently `packer init` is only in charge of installing packer
+plugins.Currently, `packer init` can only fetch binaries from public projects on
+*Github*. Github's public API, [limits the number of unauthenticated requests
+per hour one IP can
+do](https://docs.github.com/en/developers/apps/rate-limits-for-github-apps#normal-user-to-server-rate-limits).
+Packer will do its best to avoid hitting those limits and in an average local
+usage this should not be an issue. Otherwise you can set the
+`PKR_GITHUB_API_TOKEN` env var in order to get more requests per hour. Go to
+your personal [access token page](https://github.com/settings/tokens) to
+generate a new token.
```

Comment:
```suggestion
The `packer init` command is used to download Packer plugin binaries. This is
the first command that should be executed when working with a new or existing
template. This command is always safe to run multiple times. Though subsequent
runs may give errors, this command will never delete anything.   

Packer does not currently have the notion of a state like Terraform has. In other words,
currently `packer init` is only in charge of installing packer plugins.   

Currently, `packer init` can only fetch binaries from public projects on *Github*. Github's public API, [limits the number of unauthenticated requests
per hour one IP can
do](https://docs.github.com/en/developers/apps/rate-limits-for-github-apps#normal-user-to-server-rate-limits).
Packer will do its best to avoid hitting those limits and in an average local
usage this should not be an issue. Otherwise you can set the
`PKR_GITHUB_API_TOKEN` env var in order to get more requests per hour. Go to
your personal [access token page](https://github.com/settings/tokens) to
generate a new token.
```